sql/src/main/java/io/crate/protocols/postgres/Portal.java

```diff
@@ -34,6 +34,9 @@
 import java.util.List;
 import java.util.concurrent.CompletableFuture;
 
+/**
+ * A portal is the entry point for submitting queries to the cluster.
```

Comment:
suggestion: ... and models the portal objects of the PostgreSQL Wire Protocol: https://www.postgresql.org/docs/current/static/protocol-flow.html